$OpenBSD: patch-lib_tmuxinator_config_rb,v 1.7 2019/12/12 16:25:24 jasper Exp $

Prevent: "tmux: unknown option -- V" and just report the latest version

Index: lib/tmuxinator/config.rb
--- lib/tmuxinator/config.rb.orig
+++ lib/tmuxinator/config.rb
@@ -61,15 +61,7 @@ module Tmuxinator
       end
 
       def version
-        if Tmuxinator::Doctor.installed?
-          tmux_version = `tmux -V`.split(" ")[1]
-
-          if tmux_version == "master"
-            TMUX_MASTER_VERSION
-          else
-            tmux_version.to_s[/\d+(?:\.\d+)?/, 0].to_f
-          end
-        end
+        TMUX_MASTER_VERSION
       end
 
       def default_path_option
